assets/js/script.js

```diff
@@ -0,0 +1,28 @@
+class Book {
+  constructor() {
+    this.bookArray = Object.entries(window.localStorage);
+    this.html = [];
+  }
+
+  addBooks() {
+    this.bookName = document.getElementById('name-book').value;
+    this.bookAuthor = document.getElementById('author-book').value;
+    localStorage.setItem(this.bookName, this.bookAuthor);
+  }
```

Comment:
- [ ] When a book is added, it should be stored in an array. To accomplish this, create a collection that keeps a list of books, then create a function to add a new book to the collection, with title and author.
HINT :
```
const books = [];

const addBook = (title, author) => {
  id = generateID(); // Optional: 
  books = books.concat({ title, author, author });
};

```

Reply:
This issue is well resolved ✔️